### PR TITLE
Makes report issue handle null round IDs and server names

### PR DESCRIPTION
**Pull request by Cyberboss**
For them git version people

### CODE REVIEW

File: interface/interface.dm [hunk 1]
```diff
@@ -67,7 +67,10 @@
 			return
 		var/static/issue_template = file2text(".github/ISSUE_TEMPLATE.md")
 		var/servername = CONFIG_GET(string/servername)
-		src << link("[githuburl]/issues/new[GLOB.round_id ? "?body=[url_encode("Issue reported from Round ID: [GLOB.round_id][servername ? " ([servername])" : ""]\n\n[issue_template]")]" : ""]")
+		if(GLOB.round_id || servername)
```
Comment (lzimann) [inline interface/interface.dm]: could do this whole thing:
```
var/href = issue_template
if(GLOB.round_id || servername)
  href = "Issue reported from [GLOB.round_id ? " Round ID: [GLOB.round_id][servername ? " ([servername])" : ""]" : servername]\n\n[issue_template]"
DIRECT_OUTPUT(src, link("[githuburl]/issues/new?body=[url_encode(href)]"
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]
what lzi said

File: interface/interface.dm [hunk 2]
```diff
@@ -67,7 +67,10 @@
 			return
 		var/static/issue_template = file2text(".github/ISSUE_TEMPLATE.md")
 		var/servername = CONFIG_GET(string/servername)
-		src << link("[githuburl]/issues/new[GLOB.round_id ? "?body=[url_encode("Issue reported from Round ID: [GLOB.round_id][servername ? " ([servername])" : ""]\n\n[issue_template]")]" : ""]")
+		var/href = issue_template
+		if(GLOB.round_id || servername)
+			href = "Issue reported from [GLOB.round_id ? " Round ID: [GLOB.round_id][servername ? " ([servername])" : ""]" : servername]\n\n[issue_template]"
+		DIRECT_OUTPUT(src, link("[githuburl]/issues/new?body=[url_encode(href)]"
```
Comment (ShizCalev) [inline interface/interface.dm]: missing brackets

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by KorPhaeron** [comment]
@Cyberboss reopen when you're ready, stale

**Comment by AnturK** [comment]
Still broken ?

File: interface/interface.dm [hunk 3]
```diff
@@ -67,7 +67,10 @@
 			return
 		var/static/issue_template = file2text(".github/ISSUE_TEMPLATE.md")
 		var/servername = CONFIG_GET(string/servername)
-		src << link("[githuburl]/issues/new[GLOB.round_id ? "?body=[url_encode("Issue reported from Round ID: [GLOB.round_id][servername ? " ([servername])" : ""]\n\n[issue_template]")]" : ""]")
+		var/href = issue_template
+		if(GLOB.round_id || servername)
+			href = "Issue reported from [GLOB.round_id ? " Round ID: [GLOB.round_id][servername ? " ([servername])" : ""]" : servername]\n\n[issue_template]"
+		DIRECT_OUTPUT(src, link("[githuburl]/issues/new?body=[url_encode(href)]")
```
Comment (ShizCalev) [inline interface/interface.dm]: one more closing bracket

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**